client/index.html

```diff
@@ -241,29 +241,28 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Notifications</h2>
 							</div>
-							<div class="col-sm-12">
+							<div class="col-sm-6">
 								<label class="opt">
 								<input id="badge" type="checkbox" name="badge">
 								Enable badge
 								</label>
-							</div>
-							<div class="col-sm-12">
```

Comment:
Wouldn't that be better to do in a further PR? I mean, the rest of the PR is definitely wanted (granted, after some more back-and-forth) but UI stuff is usually more opinionated and might delay, even for a few days. I prefer small and split PRs that can be merged quickly, avoiding author frustration :-)
But your call if you think it cannot be split.